src/configs/chalk.c42spec

```diff
@@ -1992,27 +1992,27 @@ end users.
     type:    string
     default: "/sys/class/dmi/id/board_vendor"
     hidden:  true
-    doc:     "Path where to check AWS board vendor if running in AWS EC2"
+    doc:     "Path where to check AWS board vendor if running in AWS, Google or Microsoft nodes"
```

Comment:
that particular value is set by AWS. see link above for the ec2 docs. do GCP/Azure set the same path?